### ISSUE TITLE
Update wx flag calls

### ISSUE THREAD

**Issue by StephenHLin** [SiEPIC/SiEPIClab#7]
The newest wx has turned previous warnings into errors preventing the software from running when drawing the UI. 
Redundant calls (such as horizontal boxes specifying itself to be horizontal), need to be removed to fix.

**Comment by chessybo** [comment]
I removed the redundant calls. Is this testing the version I pushed?

**Comment by StephenHLin** [comment]
Yes, this version includes the removals you did, but not the changes for converting to python3. We have not gotten around to testing that yet. We will merge once someone in the lab tries it out.

**Comment by chessybo** [comment]
I was able to get the UI to boot. Are you sayingthere are still more redundant calls to find? That I didn't remove them all? I just kept removing them until the UI would boot without error (but not do anything)

**Comment by StephenHLin** [comment]
> I was able to get the UI to boot. Are you sayingthere are still more redundant calls to find? That I didn't remove them all? I just kept removing them until the UI would boot without error (but not do anything)

No, your version seems to have removed the redundant calls, but you also have other modifications that were for converting to python3 (such as changes to the print statement). Because no one has done a full code review followed by testing it on hardware, it would be risky to merge right away. 
At the moment we are not sure if the laser drivers and other functionalities of the hardware will work in python3. 

There are also changes that seem weird, and thus the whole code would need to be checked:
An example of this I can see is in `hp816x_N77Det_instr.py` on line 168. The original code is:
`for x in repeat(maxPWMPointsTrunc, numFullScans):`
while your new change is:
`for x in operator.mul(maxPWMPointsTrunc, numFullScans):`
I don't think itertools.repeat(A,B) and operator.mul(A,B) do the same thing. 
`Repeat` generates a list containing item A, B number of times. (https://docs.python.org/3/library/itertools.html)
`Operator.mul` will multiply A and B together.  (https://docs.python.org/3/library/operator.html)
  
As a short term fix, I uploaded a version with a fix that we had running since last November without any problems as a temporary measure. This version still is limited to python2 and ONLY includes the wxflag fixes. This is to prevent risk for other labs who are using the code.

Since only Lukas has merge privileges, to prevent confusion for him, I created this new issue to split up the 2 problems at hand (wxflags redundancies, and converting to python3).
I imagine the merging timeline would be:
merge #9 -> test #5 in a lab setting -> merge #5 and start using python3
